editor.planx.uk/src/ui/RichTextInput.tsx

```diff
@@ -1,6 +1,5 @@
 import "./RichTextInput.css";
 
-import IconButton from "@material-ui/core/IconButton";
```

Comment:
This was the problem line that broke how buttons appeared, as it was importing Material UI v4 styles which were being re-applied to buttons.